app/src/main/java/io/github/gsantner/memetastic/activity/MainActivity.java

```diff
@@ -145,7 +146,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Setup _toolbar
         setSupportActionBar(_toolbar);
-        ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(this, _drawer, _toolbar, R.string.main__navdrawer__open, R.string.main__navdrawer__close);
+        ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(this, _drawer, _toolbar,
```

Comment:
is this needed? as initial check is applied below and called in adapter too

Reply:
Would limit the check to the adapter and asset update broadcast then